lib/api_repo.js

```diff
@@ -204,6 +204,13 @@ ApiRepo.prototype.buildPackages =
               'packageInfo': that.packageInfo,
               'generated': generated
             }, templateInfo[l]);
+            if (l == 'nodejs' && !that.opts.nodejsUsePbjs) {
+              opts.templates = ['service.js.mustache'];
```

Comment:
I'm trying to understand how this works. Does setting this override the templates in `packager.js`? If so, could you add a comment? Otherwise, it's a little hard to see, since this is the only template specified in `api_repo.js`, while the rest are in `packager.js`.